pglookout/cluster_monitor.py

```diff
@@ -201,6 +224,10 @@ def _query_cluster_member_state(self, instance, db_conn):
                 master_position = c.fetchone()
                 maybe_standby_result["pg_last_xlog_replay_location"] = master_position["pg_last_xlog_replay_location"]
                 f_result = maybe_standby_result
+
+                if db_conn.server_version >= 100000:
```

Comment:
PG < 10 has been EOLed for a while, that's why I didn't bother with anything older here.